docs/src/components/landing/ToolpadHero.js

```diff
@@ -17,20 +21,39 @@ export default function ToolpadHero() {
             display: 'flex',
             alignItems: 'center',
             justifyContent: { xs: 'center', md: 'center' },
-            '& > *': { mr: 1, width: 28, height: 28 },
+            '& > *': { mr: 1, width: 84, height: 28 },
           }}
         >
-          <IconImage name="product-toolpad" /> Toolpad
+          <IconImage name="product-toolpad" /> <GradientText>MUI Toolpad</GradientText>
         </Typography>
         <Typography variant="h1" sx={{ my: 2 }}>
           Low-code
           <br />
           <GradientText>admin builder</GradientText>
         </Typography>
-        <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500, mx: 'auto' }}>
-          Drag and drop pre-built components, connect to data sources, APIs and build your internal
+        <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 520, mx: 'auto' }}>
+          Drag and drop pre-built components, connect to data sources and APIs, and build your internal
           tools 10x faster. Open-source and powered by MUI.
         </Typography>
+        <Typography color="text.secondary" sx={{ fontWeight: 700, mb: 2, mx: 'auto' }}>
+          Get notified of the beta release.
+        </Typography>
+        <TextField
+          variant="outlined"
+          placeholder="example@email.com"
+          sx={{ mr: 2, mb: { xs: 2, sm: 0 }, width: { xs: '100%', sm: 'auto' } }}
+        />
+        <Button
+          component={Link}
+          href={'/'}
+          noLinkStyle
+          size="large"
+          variant="contained"
+          endIcon={<KeyboardArrowRightRounded />}
+          sx={{ width: { xs: '100%', sm: 'auto' } }}
+        >
+          Sign up
+        </Button>
```

Comment:
there is a size issue on this one:

<img width="368" alt="Screenshot 2022-08-12 at 12 58 12" src="https://user-images.githubusercontent.com/3165635/184409940-f4a18f9d-1e03-4c00-9948-aaccedfb108b.png">